.changeset/cyan-drinks-enjoy.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/admin": major
+---
+
+Change theming method of `Menu`
+
+-   Rename `permanent` class key to `permanentDrawer` and `temporary` class key to `temporaryDrawer`
+-   Delete `permanentDrawerProps` and `temporaryDrawerProps`, use `slotProps` instead
+-   Instead of using the `style` prop for `PaperProps` in PermanentDrawer and TemporaryDrawer, use the `sx` prop.
```

Comment:
I don't think this line is necessary. 

When passing in a value for `style` through `temporaryDrawerPaperProps` or `permanentDrawerPaperProps` it will technically still work, we only changed it to use `sx` internally. 

```suggestion
```